src/core/api/thread_api.cpp

```diff
@@ -97,6 +97,15 @@ otError otThreadGetLeaderRloc(otInstance *aInstance, otIp6Address *aLeaderRloc)
     return instance.Get<Mle::MleRouter>().GetLeaderAddress(*static_cast<Ip6::Address *>(aLeaderRloc));
 }
 
+otError otThreadGetLeaderAloc(otInstance *aInstance, otIp6Address *aLeaderAloc)
+{
+    Instance &instance = *static_cast<Instance *>(aInstance);
+
+    assert(aLeaderAloc != NULL);
```

Comment:
in the function declaration, here should return `OT_ERROR_INVALID_ARGS`?
https://github.com/openthread/openthread/pull/4268/files#diff-e01913b0351246189e44602cc26bcfabR338

Reply:
Good catch!

Reply:
We have same issue for `otThreadGetLeaderRloc` and `otThreadGetNextNeighborInfo`. I will just fix `otThreadGetLeaderAloc` here.